core/unix/module_elf.h

```diff
@@ -42,6 +42,11 @@
 #    define DT_RELR 36
```

Comment:
> iX: RHEL-7 build fix due to EM_RISCV.

style: We don't put `iX: ` in the title: just "RHEL-7 build fix due to EM_RISCV".

Reply:
Fixed.